test/parallel/test-cluster-disconnect.js

```diff
@@ -72,7 +73,11 @@ if (cluster.isWorker) {
     let online = 0;
 
     for (let i = 0, l = workers; i < l; i++) {
-      cluster.fork().on('listening', common.mustCall(() => {
+      cluster.fork().on('listening', common.mustCall((address) => {
+        if (server_ports.indexOf(address.port) === -1) {
```

Comment:
Nit: if this change will not be backported to v4 we can use `Array.prototype.includes()` instead of `Array.prototype.indexOf()`.